core/src/main/java/org/mskcc/cbio/portal/scripts/ImportExtendedMutationData.java

```diff
@@ -431,41 +431,48 @@ public void importData() throws IOException, DaoException {
             }
         }
 
-                for (MutationEvent event : newEvents) {
-                    try {
-                        DaoMutation.addMutationEvent(event);
-                    } catch (DaoException ex) {
-                        throw ex;
-                    }
-                }
+        for (MutationEvent event : newEvents) {
+            try {
+                DaoMutation.addMutationEvent(event);
+            } catch (DaoException ex) {
+                throw ex;
+            }
+        }
 
-                for (ExtendedMutation mutation : mutations.values()) {
-                    try {
-                        DaoMutation.addMutation(mutation,false);
-                    } catch (DaoException ex) {
-                        throw ex;
-                    }
-                }
+        for (ExtendedMutation mutation : mutations.values()) {
+            try {
+                DaoMutation.addMutation(mutation,false);
+            } catch (DaoException ex) {
+                throw ex;
+            }
+        }
+
+        ClinicalAttribute attr = new ClinicalAttribute("MUTATION_COUNT", "Mutation Count", "Mutation Count", "NUMBER",
+            false, "1", geneticProfile.getCancerStudyId());
```

Comment:
@zhx828 what's the priority for "MUTATION_COUNT" in the study view?

Reply:
https://github.com/cBioPortal/cbioportal/blob/master/portal/src/main/webapp/js/src/dashboard/resources/configs.json#L28 seems like it's 30